mathgenerator/mathgen.py

```diff
@@ -713,6 +713,27 @@ def fourthAngleOfQuadriFunc(maxAngle = 180):
     problem = f"Fourth angle of quadrilateral with angles {angle1} , {angle2}, {angle3} ="
     solution = angle4
     return problem, solution
+def confidenceIntervalFunc():
+    n=random.randint(20,40)
+    j=random.randint(1,4)
+    lst=random.sample(range(200,300),n)
+    lst_per=[80 ,90, 95, 99]
+    lst_t = [1.282, 1.645, 1.960, 2.576]
+    mean=0
+    sd=0
+    for i in lst:
+        count= i + mean
+        mean=count
+    mean = mean/n
+    for i in lst:
+        x=(i-mean)**2+sd
+        sd=x
+    sd=sd/n
+    standard_error = lst_t[j]*math.sqrt(sd/n)
```

Comment:
Your program gave the following error when testing:
```
  File "mathgen.py", line 732, in confidenceIntervalFunc
    standard_error = lst_t[j]*math.sqrt(sd/n)
IndexError: list index out of range
```
This occurred about every other time the program was run.

Reply:
@lukew3 Done can you review again?  :)